bokehjs/src/coffee/models/annotations/color_bar.coffee

```diff
@@ -37,6 +38,9 @@ export class ColorBarView extends AnnotationView
     return {x: x, y: -y}
 
   _get_size: () ->
+    if not @model.color_mapper?
```

Comment:
This is a recurring change, because `color_mapper` doesn't have a default value. This needs to be improved in a future PR. For now this is just to make tests run.